index.html

```diff
@@ -3,19 +3,88 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <title>Great Idea! - Fully Responsive!</title>
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
```

Comment:
You need to have `<body>` tags in your html file. While it's recommended to have a containing `div` that wraps all other content, wrapping _that_ div should always be a `body` tag